### PR TITLE
No pm role

### PR DESCRIPTION
**Pull request by avarabyeu**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/core/item/DeleteTestItemHandlerImpl.java [hunk 1]
```diff
@@ -147,7 +146,7 @@ private void validateRoles(TestItem testItem, User user, Project project) {
 			 * Only LEAD and PROJECT_MANAGER roles could delete testItems
```
Comment (pictureb00k) [inline src/main/java/com/epam/ta/reportportal/core/item/DeleteTestItemHandlerImpl.java]: remove lead

File: src/main/java/com/epam/ta/reportportal/core/item/UpdateTestItemHandlerImpl.java [hunk 2]
```diff
@@ -238,7 +239,7 @@ private TestItem validate(String projectName, String userName, String id) {
 		String launchOwner = launch.getUserRef();
 		if (userRepository.findOne(userName).getRole() != UserRole.ADMINISTRATOR) {
 			expect(projectName, equalTo(project.getName())).verify(ACCESS_DENIED);
-			if (project.getUsers().containsKey(userName) && project.getUsers().get(userName).getProjectRole().getRoleLevel() < 2) {
+			if (project.getUsers().containsKey(userName) && project.getUsers().get(userName).getProjectRole().sameOrHigherThan(ProjectRole.PROJECT_MANAGER)) {
```
Comment (pictureb00k) [inline src/main/java/com/epam/ta/reportportal/core/item/UpdateTestItemHandlerImpl.java]: lower than?

File: src/main/java/com/epam/ta/reportportal/core/launch/impl/FinishLaunchHandler.java [hunk 3]
```diff
@@ -207,7 +206,7 @@ private Project validateRoles(Launch launch, String userName, String projectName
 			 * Only LEAD and PROJECT_MANAGER roles could delete launches
```
Comment (pictureb00k) [inline src/main/java/com/epam/ta/reportportal/core/launch/impl/FinishLaunchHandler.java]: remove lead

File: src/main/java/com/epam/ta/reportportal/core/item/DeleteTestItemHandlerImpl.java (hunk 1 above)
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/core/item/DeleteTestItemHandlerImpl.java]: Fixed.

File: src/main/java/com/epam/ta/reportportal/core/launch/impl/FinishLaunchHandler.java (hunk 3 above)
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/core/launch/impl/FinishLaunchHandler.java]: Fixed.

**Pull request merged**